src/es6-import-test.js

```diff
@@ -0,0 +1,3 @@
+import canonicalGlobalNav from '../dist/index';
```

Comment:
Does `canonicalGlobalNav` relate to the name you gave to it in the `output` definition of the `rollup.config.js`. I just tried this out in another test repo and had to use `import * as globalNav from './global'` to get it to work.

Maybe we could have the different use cases tested in their own folder? Maybe a bit overkill, since we are testing rollup?

Reply:
I'm not exactly sure how rollup handles named exports. They have a [section about it in the docs](https://github.com/rollup/rollup-plugin-commonjs#custom-named-exports) but I couldn't get it work in this example.